modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,251 @@ It also includes enhancements to the Eventing and Search services.
+|===
+| Issue | Description
+
+
+| https://issues.couchbase.com/browse/MB-33696[MB-33696^]
+| *Summary*: The analytics service on one node failed to startup repeatedly due to a failure while performing a rollback and threw an exception. This has been fixed.
+
+| https://issues.couchbase.com/browse/MB-33595[MB-33595^]
+| *Summary*: Variables bound by a QuantifiedExpression were incorrectly included when computing free variables for that expression. This has been fixed and these variables will now be excluded from the free variable set.
+
+| https://issues.couchbase.com/browse/MB-33564[MB-33564^]
+| *Summary*: Fixed an issue where the predicate in the WHERE clause was incorrectly pushed through the running aggregate operator (AT subclause), which then lead to wrong results.
+
+| https://issues.couchbase.com/browse/MB-33338[MB-33338^]
+| *Summary*: It was observed that a rebalance in of new analytics nodes hangs indefinitely on a firewall misconfiguration where newly added nodes were unable to contact the CC. This has been fixed so that rebalance fails if the Keep-Nodes do not join within a reasonable timeframe (2 minutes).
+
+| https://issues.couchbase.com/browse/MB-33025[MB-33025^]
+| *Summary*: Fixed an issue where the analytics HTTP server incorrectly assumed that all request payloads are UTF-8 encoded. The analytics HTTP responses are encoded using system-default encoding and the service will now attempt to honor any requested encodings (Accept-Charset), or default to UTF-8.
+
+| https://issues.couchbase.com/browse/MB-33016[MB-33016^]
+| *Summary*: Fixed an internal compiler error caused by index selection when the search expression was not constant.
+
+| https://issues.couchbase.com/browse/MB-32777[MB-32777^]
+| *Summary*: Fixed an issue where array functions, most notably the array_intersect() function, caused an out of memory exception due to object creations, especially for large datasets.
+
+| https://issues.couchbase.com/browse/MB-32775[MB-32775^]
+| *Summary*: Fixed an issue to catch all unexpected exceptions on network threads and close the connection in such cases to avoid attempting to process the message causing the exception forever.
+
+| https://issues.couchbase.com/browse/MB-32745[MB-32745^]
+| *Summary*: The analytics service now uses the Couchbase temp directory, which is set to `/var/lib/couchbase/tmp` by default, and no longer requires access to the operating system's temp directory. 
+|===
+
+==== Cross Datacenter Replication (XDCR)
+
+[#table_fixedissues_v602-xdcr,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-33970[MB-33970^]
+| *Summary*: By default, XDCR will prevent replications from being created on a source ephemeral bucket with NRU policy. This release of Couchbase Server introduces a special XDCR internal settings flag, `AllowSourceNRUCreation`, that allows replications to be created on a source ephemeral bucket with NRU policy. 
```

Comment:
> This release of Couchbase Server introduces

This *specific* release of Couchbase Server *includes* (?)

Introduces means that it's present in all future releases, and that's not the case as 6.5 will not have this flag. (But 6.0.3, if comes out in the future, will have it)

Reply:
Yes, I see your point. Updating the text to say "includes" instead of "introduces". Thanks.